RadixWallet/Features/SettingsFeature/DisplayMnemonics/Coordinator/DisplayMnemonics.swift

```diff
@@ -12,6 +12,8 @@ public struct DisplayMnemonics: Sendable, FeatureReducer {
 		public var destination: Destination.State? = nil
 
 		public var deviceFactorSources: IdentifiedArrayOf<DisplayEntitiesControlledByMnemonic.State> = []
+		fileprivate var problems: [SecurityProblem]?
+		fileprivate var entities: IdentifiedArrayOf<EntitiesControlledByFactorSource>?
```

Comment:
I think `entities` is a misleading name. A collection of accounts, or a collection of personas or a mix of both those would be `entities`. This a collection of Tuples with `(FactorSource, Entities)`. 

`factorsWithEntitiesTheyControl` is more accurate and maybe not too long?

Reply:
then we should rename `EntitiesControlledByFactorSource` as well?